kroxylicious-app/pom.xml

```diff
@@ -192,6 +196,9 @@
                             <descriptors>
                                 <descriptor>src/assembly/binary-distribution.xml</descriptor>
                             </descriptors>
+                            <descriptorRefs>
+                                <descriptorRef>jar-with-dependencies</descriptorRef>
```

Comment:
I've not objection to including this but I'm curious to understand why this was needed.

Reply:
Without it I received errors from log4j complaining about disruptor not being on the classpath when running Kroxylicious from image. This solved that problem.

Reply:
Interesting I, I should go check what it actually does.

Reply:
I tried reverting this descriptorRef change and it appears to work for me, `disruptor` jar is in the distribution libs dir and image works okay. To test I ran:

```
PUSH_IMAGE=y REGISTRY_DESTINATION=quay.io/${MY_QUAY_ORG}/kroxylicious ./scripts/build-image.sh                                                                                    
KROXYLICIOUS_IMAGE=quay.io/${MY_QUAY_ORG}/kroxylicious:0.9.0-SNAPSHOT ./scripts/run-example.sh kubernetes-examples/network-topologies/portperbroker_plain/
```